ush/forecast_predet.sh

```diff
@@ -568,7 +568,7 @@ CICE_predet(){
 
   # CICE does not have a concept of high frequency output like FV3
   # Convert output settings into an explicit list for CICE
-  CICE_OUTPUT_FH=$(seq -s ' ' "${FHMIN}" "${FHOUT_OCNICE}" "${FHMAX}")
+  CICE_OUTPUT_FH=$(seq -s ' ' "${FHMIN}" "${FHOUT_ICE}" "${FHMAX}")
```

Comment:
I think we can avoid the complicated changes later on by just generating the output hours correctly up-front. This is also more general than handling just the 24-h case.
```suggestion
  ((offset = ( cyc + FHMIN ) % FHOUT_ICE))
  if (( offset == 0 )); then
    CICE_OUTPUT_FH=($(seq -s ' ' "${FHMIN}" "${FHOUT_ICE}" "${FHMAX}"))
  else
    CICE_OUTPUT_FH=("${FHMIN}")
    CICE_OUTPUT_FH+=($(seq -s ' ' "$((FHMIN+offset))" "${FHOUT_ICE}" "${FHMAX}"))
    CICE_OUTPUT_FH+=("${FHMAX}")
  fi
```

Reply:
Yes, this seems like a less complicated and more generalized approach. These changes have just been applied locally. I am retesting to make sure there are no issues. Thanks.

Reply:
@WalterKolczynski-NOAA I just finished a test with this suggestion and it works great for the most part. The only issue is that the first lead time (f012) is not being picked up by the global-workflow because the CICE output is using "24h" in its filename, even though it is actually a 12 hour average. Please see this screenshot of an `ls` in `${RUNDIRS}/CICE_OUTPUT`.  
![Screenshot from 2024-06-04 18-43-50](https://github.com/NOAA-EMC/global-workflow/assets/48259628/7e4fa585-41ba-4bb1-92f7-49c84ee3a53e)

Reply:
See below suggestion in postdet to fix this.